_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -0,0 +1,723 @@
+access, and no other groups. All Gruntwork subscribers have access to the core modules, which reside in
+Gruntwork's link:https://github.com/gruntwork-io/toc[infrastructure as code repositories].
+
+[[wrapper_modules]]
+=== Compliance wrapper modules
+The compliance wrapper modules are an extension of the IaC Library. They use the
+link:https://www.terraform.io/docs/modules/sources.html[`source` argument in a Terraform module block] to invoke
+the core module with a configuration that is customized for compliance with the CIS AWS Foundations Benchmark.
+These modules are in the link:https://github.com/gruntwork-io/cis-compliance-aws[`cis-compliance-aws`
+repository] (accessible to Gruntwork Compliance subscribers).
+
+[[infrastructure_modules]]
+=== infrastructure-modules
+The `infrastructure-modules` are your organization's "blueprint" for how to deploy infrastructure. You can
+use `infrastructure-modules` to customize the settings according to the needs of your environment. Subscribers
+can refer to the
+canonical link:https://github.com/gruntwork-io/infrastructure-modules-multi-account-acme[ACME infrastructure-modules
+reposistory] for an example of how you might use `infrastructure-modules`.
+
+If you're using Terragrunt, the `infrastructure-modules` are optional; you can call the compliance
+wrapper modules directly as the `source` from a Terragrunt configuration. The benefit of this is that you have less code to maintain by depending directly on Gruntwork's upstream compliance modules.
+
+[[infrastructure_live]]
+=== infrastructure-live
+`infrastructure-live` uses link:https://github.com/gruntwork-io/terragrunt[Terragrunt] to make it easier to
+work with Terraform modules in multiple environments. `infrastructure-live` is optional - you can use all of the modules
+with or without Terragrunt.
+
+If you're not using Terragrunt, you can use `infrastructure-modules` to call the compliance
+wrapper modules. Refer to the canonical link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme[ACME
+infrastructure-live repository], and in particular the
+link:https://github.com/gruntwork-io/infrastructure-live-multi-account-acme/tree/master/_docs[Infrastructure
+walkthrough] for comprehensive documentation on how `infrastructure-live`, `infrastructure-modules`, and the core IaC
+modules interact.
+ +
+[[benefits]]
+=== Benefits
+This modular, decomposed approach allows for maximum code reuse. The core modules can be used with or without
+compliance, depending on how they are configured. The compliance wrappers are like shadows of the core
+modules; they pass through most of the variables to the core modules without alteration, but hard code any
+settings needed for compliance. When you call the compliance modules from your own code in
+`infrastructure-modules`, you only need to set up any variables that are custom for your environment. Often
+times the default settings are good enough.
```

Comment:
Should customers use the CIS wrappers in pre-prod envs or only prod?

Reply:
Clarified this:
> You can use this approach on AWS account. In many cases, you'll only need compliance for production accounts, but the same methodology can be applied to pre-production accounts as well.